wftests/warrior_tests/testcases/common_actions_tests/tc_verify_arith_exp.xml

```diff
@@ -0,0 +1,204 @@
+                </step>
+                <step Driver="common_driver" Keyword="verify_arith_exp" TS="9" draft="no">
+                        <Arguments>
+                                <argument name="expected" value="41"/>
+                                <argument name="comparison" value="lt"/>
+                                <argument name="expression" value="${ENV.var1}+${REPO.var2}+10"/>
+                        </Arguments>
+                        <onError action="next"/>
+                        <Description>Verify arithmetic expression: lt</Description>
+                        <iteration_type type=""/>
+                        <Execute ExecType="Yes">
+                                <Rule Condition="" Condvalue="" Else="" Elsevalue=""/>
+                        </Execute>
+                        <context>positive</context>
+                        <impact>impact</impact>
+                        <runmode type="Standard" value=""/>
+                        <Iteration_type type="Standard"/>
+                </step>
+                <step Driver="common_driver" Keyword="verify_arith_exp" TS="10" draft="no">
+                        <Arguments>
+                                <argument name="expected" value="39"/>
+                                <argument name="comparison" value="le"/>
+                                <argument name="expression" value="${ENV.var1}+${REPO.var2}-1+11"/>
+                        </Arguments>
+                        <onError action="next"/>
+                        <Description>Verify arithmetic expression: le</Description>
+                        <iteration_type type=""/>
+                        <Execute ExecType="Yes">
+                                <Rule Condition="" Condvalue="" Else="" Elsevalue=""/>
+                        </Execute>
+                        <context>negative</context>
+                        <impact>impact</impact>
+                        <runmode type="Standard" value=""/>
+                        <Iteration_type type="Standard"/>
+                </step>
+                <step Driver="common_driver" Keyword="verify_arith_exp" TS="11" draft="no">
+                        <Arguments>
+                                <argument name="expected" value="39"/>
+                                <argument name="comparison" value="le"/>
+                                <argument name="expression" value="ls -lrt /*"/>
+                        </Arguments>
+                        <onError action="next"/>
+                        <Description>User Expression is evaluated before processing with Python EVAL, Expect ERROR status</Description>
+                        <iteration_type type=""/>
+                        <Execute ExecType="Yes">
+                                <Rule Condition="" Condvalue="" Else="" Elsevalue=""/>
+                        </Execute>
+                        <context>negative</context>
+                        <impact>noimpact</impact>
```

Comment:
For expected Error status
Please use verify_data in common_actions to verify the step_#_result string is error

Reply:
Done